src/crate/client/tests.py

```diff
@@ -250,8 +247,14 @@ def tearDownWithCrateLayer(test):
     # clear testing data
     conn = connect(crate_host)
```

Comment:
you could use with ``connect(crate_host) as conn`` to avoid the try/except stuff below